### PR TITLE
Add new commands to cli

### PR DESCRIPTION
**Pull request by grimadas**
* Update pictures
* Add minor grammar fixes
* Add update to flag names

### CODE REVIEW

File: source/includes/_cli.md [hunk 1]
```diff
@@ -11,51 +11,52 @@ To generate files containing private and public keys `--new_account` flag should
...
 
 ![image](../images/cli/image.png)
 
 ## Transaction CLI
 
 Start up of transaction cli will trigger the creation of a new transaction.
-Each transaction consists of commands, being less or equal than **65000**. User is offered to add commands to new formed transaction.
-All meta data of transaction will be filled automatically (signature, tx_counter, creator account, timestamp).
-Currently following Iroha commands are supported in Interactive CLI:
+Each transaction consists of commands, being less or equal than **65000**. User is offered to add commands to anew transaction.
```
Comment (kamilsa) [inline source/includes/_cli.md]: equal than? Better to say being equal or less than ...

**Review by kamilsa — CHANGES_REQUESTED** [review]

File: source/includes/_cli.md (hunk 1 above)
Comment (Warchant) [reply source/includes/_cli.md]: to **anew**

**Comment by Warchant** [comment]
Also, use terminal when you make a commit.
write 
```
git commit -s 
```

and you will never get DCO error again.

File: source/includes/_cli.md (hunk 1 above)
Comment (grimadas) [reply source/includes/_cli.md]: anew is a legit word

**Review by Warchant — APPROVED** [review]

**Review by kamilsa — APPROVED** [review]

**Pull request merged**